flexbox/src/main/res/layout/activity_minwidth_test.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please put this file under flexbox/src/androidTest/res
This file is used only for testing.